elephant/statistics.py

```diff
@@ -389,39 +460,44 @@ def lv(time_intervals, with_nan=False):
 
 
 @deprecated_alias(v='time_intervals')
-def cv2(time_intervals, with_nan=False):
+def lvr(time_intervals, R=5*pq.ms, with_nan=False):
```

Comment:
deprecated aliases are made for existing functional

don't use them while adding a new function; otherwise, we will never get rid of them

Reply:
Hey!
Right, I just did not know why that was there so I copied it over again hehe
Should be fine now